### PR TITLE
AKU-953: SelectedItemStateMixin fix

### PR DESCRIPTION
**Pull request by draperd**
This PR addresses https://issues.alfresco.com/jira/browse/AKU-953 to ensure that the SelectedItemStateMixin treats all values appropriately. A new unit test has been added to verify and prevent regressions.

### CODE REVIEW

File: aikau/src/test/resources/alfresco/renderers/SelectorTest.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * This test renders examples of FileTypes.
+ *
+ * The test is simple and much of its validity is in the use of slightly damaged or incomplete models to inspect edge cases.
+ *
+ * @author Richard Smith
+ */
```
Comment (MartinDoyleUK) [inline aikau/src/test/resources/alfresco/renderers/SelectorTest.js]: This comment block seems wrong??

File: aikau/src/test/resources/config/Suites.js [hunk 2]
```diff
@@ -252,6 +252,7 @@ define(function() {
       "alfresco/renderers/PublishPayloadMixinOnActionsTest",
       "alfresco/renderers/ReorderTest",
       "alfresco/renderers/SearchResultPropertyLinkTest",
+      "src/test/resources/alfresco/renderers/SelectorTest",
```
Comment (MartinDoyleUK) [inline aikau/src/test/resources/config/Suites.js]: I didn't explicitly mention it (apologies) but I discovered that we don't need the src/test prefix, and so it was removed from all tests.

File: aikau/src/test/resources/testApp/WEB-INF/classes/alfresco/site-webscripts/alfresco/renderers/Selector.get.desc.xml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<webscript>
+  <shortname>Selector Renderer</shortname>
+  <description>Displays Selector renderers for testing.</description>
+  <description></description>
```
Comment (MartinDoyleUK) [inline aikau/src/test/resources/testApp/WEB-INF/classes/alfresco/site-webscripts/alfresco/renderers/Selector.get.desc.xml]: Duplicate element
